src/data/languages/Spanish.json

```diff
@@ -0,0 +1,444 @@
+{
+	"info": {
+		"language": "Español",
+		"language_english": "Spanish",
+		"fallback": ["English"],
+		"maintainer": "MrJako2001",
+		"maintainer_link": "https://anilist.co/user/MrJaco/",
+		"discussion_link": "",

```

Comment:
Add https://github.com/hohMiyazawa/Automail/issues/69 as the discussion link if you don't want to manage your own discussion place.

Reply:
Fixed